tools/hammer_loop.sh

```diff
@@ -42,7 +42,7 @@ else
     rm -r "$TEST_ROOT"
 fi
 REGION_ROOT=${REGION_ROOT:-/var/tmp}
-MY_REGION_ROOT=${REGION_ROOT/hammer_loop}
```

Comment:
This is wrong, and should be `MY_REGION_ROOT=${REGION_ROOT}/hammer_loop`
Should be fixed in newer commits.